internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -272,51 +267,53 @@ func (opts *Opts) newCluster() *atlas.Cluster {
 		},
 	}
 
+	if opts.providerName() != tenant {
+		diskSizeGB := atlas.DefaultDiskSizeGB[strings.ToUpper(opts.providerName())][opts.tier]
+		mdbVersion, _ := cli.DefaultMongoDBMajorVersion()
+		cluster.DiskSizeGB = &diskSizeGB
+		cluster.MongoDBMajorVersion = mdbVersion
+	}
+
```

Comment:
I had to add this piece of code to prevent these errors

```
Error: POST https://cloud-dev.mongodb.com/api/atlas/v1.5/groups/5e4e593f70dfbf1010295836/clusters: 400 (request "TENANT_ATTRIBUTE_READ_ONLY") The attribute diskSizeGB is read-only for tenant clusters and cannot be changed by the user.

Error: POST https://cloud-dev.mongodb.com/api/atlas/v1.5/groups/5e4e593f70dfbf1010295836/clusters: 400 (request "TENANT_ATTRIBUTE_READ_ONLY") The attribute mongoDBMajorVersion is read-only for tenant clusters and cannot be changed by the user.
```